src/utils/TextFormat.php

```diff
@@ -130,7 +130,7 @@ private static function preg_replace(string $pattern, string $replacement, strin
 	 * @return string[]
 	 */
 	public static function tokenize(string $string) : array{
-		$result = preg_split("/(" . TextFormat::ESCAPE . "[0-9a-fk-or])/u", $string, -1, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE);
+		$result = preg_split("/(" . TextFormat::ESCAPE . "[0-9a-fg-or])/u", $string, -1, PREG_SPLIT_NO_EMPTY | PREG_SPLIT_DELIM_CAPTURE);
```

Comment:
Now you broke it. It should be `a-gk-o`.

Reply:
xd, sorry im tierd af

Reply:
2days up

Reply:
@xxAROX You forgot to fix this line. You only fixed line 147.

Reply:
> 

thanks for notice, i didn't saw that one